README.md

```diff
@@ -4,9 +4,16 @@ Here you will find any logos/banners/art we have had made for PolyKit! Most item
 <br />
 
 # Logos
-![Circular](/logos/circular.svg)
-![Square](/logos/square.svg)
-![Transparent](/logos/transparent.svg)
+
+<div align="center" width="100%">
+
+<img src="/logos/circular.svg" alt="Circular" width="30%"/>
+
+<img src="/logos/transparent.svg" alt="Transparent" width="30%"/>  
+
+<img src="/logos/square.svg" alt="Square" width="30%"/>
```

Comment:
Instead of centering can we do like, `max-width: 150px`. I couldn't figure out how to make them all on the same line :joy: thanks for the pr!